hack/make-rules/test-cmd.sh

```diff
@@ -2324,6 +2324,26 @@ __EOF__
   kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.unschedulable}}" '<no value>'
 
 
+
+  ########################
+  # authorization.k8s.io #
+  ########################
+
+  # check remote authorization endpoint, kubectl doesn't actually display the returned object so this isn't super useful
+  # but it proves that works
```

Comment:
Just thinking aloud here: If we had called these request and response instead of spec and status, then we could have had a convention in kubectl that if a thing has request and response, then kubectl should print out the response.